apps/backend/src/eventHandlers/email/eliEmailHandler.ts

```diff
@@ -0,0 +1,324 @@
+              },
+            };
+          }),
+        ],
+      };
+
+      mailService
+        .sendMail(options)
+        .then((res: any) => {
+          logger.logInfo('Emails sent on proposal submission:', {
+            result: res,
+            event,
+          });
+        })
+        .catch((err: string) => {
+          logger.logError('Could not send email(s) on proposal submission:', {
+            error: err,
+            event,
+          });
+        });
+
+      return;
+    }
+
+    case Event.USER_CREATED: {
+      if (process.env.NODE_ENV === 'development') {
+        //await userDataSource.setUserEmailVerified(
+        //    event.userlinkresponse.user.id
+        //);
+        logger.logInfo('Set user as verified without sending email', {
+          userId: event.userlinkresponse.user.id,
+          event,
+        });
+      } else {
+        mailService
+          .sendMail({
+            content: {
+              template_id: 'user-office-account-verification',
+            },
+            substitution_data: {
+              title: 'ESS User portal verify account',
+              buttonText: 'Click to verify',
+              link: event.userlinkresponse.link,
+            },
+            recipients: [{ address: event.userlinkresponse.user.email }],
+          })
+          .then((res: any) => {
+            logger.logInfo('Email sent on user creation:', {
+              result: res,
+              event,
+            });
+          })
+          .catch((err: string) => {
+            logger.logError('Could not send email on user creation:', {
+              error: err,
+              event,
+            });
+          });
+      }
+
+      return;
+    }
+    case Event.PROPOSAL_NOTIFIED: {
+      const principalInvestigator = await userDataSource.getUser(
+        event.proposal.proposerId
+      );
+      if (!principalInvestigator) {
+        return;
+      }
+      const { finalStatus } = event.proposal;
+      let templateId = '';
+      if (finalStatus === ProposalEndStatus.ACCEPTED) {
+        templateId = 'Accepted-Proposal';
+      } else if (finalStatus === ProposalEndStatus.REJECTED) {
+        templateId = 'Rejected-Proposal';
+      } else if (finalStatus === ProposalEndStatus.RESERVED) {
+        templateId = 'Reserved-Proposal';
+      } else {
+        logger.logError('Failed email notification', { event });
+
+        return;
+      }
+
+      mailService
+        .sendMail({
+          content: {
+            template_id: templateId,
+          },
+          substitution_data: {
+            piPreferredname: principalInvestigator.preferredname,
+            piLastname: principalInvestigator.lastname,
+            proposalNumber: event.proposal.proposalId,
+            proposalTitle: event.proposal.title,
+            commentForUser: event.proposal.commentForUser,
+          },
+          recipients: [
+            { address: principalInvestigator.email },
+            /*{
```

Comment:
This comment here as well can be cleaned up

Reply:
Fixed, thanks.